systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeST.java

```diff
@@ -208,6 +215,20 @@ void testUpgradeKafkaWithoutVersion() throws IOException {
                 .stream().filter(c -> c.getName().equals("kafka")).findFirst().get().getImage(), containsString("2.6.0"));
     }
 
+    void applyKafkaWithoutVersion(File kafka, String clusterName) throws IOException {
+        YAMLMapper mapper = new YAMLMapper();
+        JsonNode node = mapper.readTree(kafka);
+        ObjectNode kafkaNode = (ObjectNode) node.at("/spec/kafka");
+        kafkaNode.remove("version");
```

Comment:
I guess you need to also change the message format version?

Reply:
I was thinking that too, but in the example there is already the `log.message.format.version` so I decided to not add it there:
```
config:
        log.message.format.version: "2.5"
        offsets.topic.replication.factor: 3
        transaction.state.log.min.isr: 2
        transaction.state.log.replication.factor: 3

```

I guess I can add it there to be sure.